dtorrent/Cargo.toml

```
@@ -13,3 +13,4 @@ rand = "0.8.5"
 bencoder = { path = "../bencoder" }
 logger = { path = "../logger"}
 url_encoder = { path = "../url_encoder" }
+tokio = { version = "1", features = ["rt-multi-thread","macros"]}
```

I usually like to keep alphabetical order along with a consistent style, especially in files that don't have a formatting tool like TOML. Therefore I'd like to make the following suggestion:
```suggestion
bencoder = { path = "../bencoder" }
logger = { path = "../logger"}
tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
url_encoder = { path = "../url_encoder" }
```

I've also noticed you're missing the final newline. While it's not mandatory, I think it's good to keep it.

Bonus tip: While developing with `tokio` (or any other `feature`d library), I think it's fine to use `features = ["full"]`. Compile times shouldn't increase that much thanks to caching, except in the CI (there's no caching by default). Since `tokio` has lots of features for everything with dependencies between them, I think it'll be fine if you use `features = ["full"]` on your branches as long as you change it back to just the required features before marking the PR as ready for review.

---

Thanks for the review. I already made the necessary chances and the bonus tip was one of my questions.